ansible_collections/arista/avd/roles/eos_designs/templates/underlay/interfaces/ethernet-interfaces.j2

```diff
@@ -15,8 +15,12 @@ ethernet_interfaces:
 {%         if p2p_uplinks_qos_profile is arista.avd.defined %}
     service_profile: {{ p2p_uplinks_qos_profile }}
 {%         endif %}
-{%         if link.ptp is arista.avd.defined %}
-    ptp: {{ link.ptp }}
+{%         if link.ptp.enable is arista.avd.defined(true) %}
+{%             set ptp_profile = switch.ptp.profile | arista.avd.default("unknown") %}
```

Comment:
is the default needed here? this should be set by eos_design_facts as soon as ptp is enabled?

Reply:
you are right, it would always have some value, so we can remove the default here (i helped add it)

Reply:
@nielsjlarsen you can just use `switch.ptp.profile` below instead of `ptp_profile`.

Reply:
fixed